### ISSUE TITLE
honesty depends on click, buit doesn't list it as a dependency

### ISSUE THREAD

**Issue by jreese** [python-packaging/honesty#2]
```
(venv37) jreese@jreese-mbp ~/scratch/aiosqlite-0.10.0 ‹130› » pip install -U honesty
Collecting honesty
  Downloading https://files.pythonhosted.org/packages/e2/ea/ad91b23740c5103f15422f1344b92ce4ccff7986ab2837ea0b4bae40dc80/honesty-0.1.1-py3-none-any.whl
Installing collected packages: honesty
Successfully installed honesty-0.1.1
(venv37) jreese@jreese-mbp ~/scratch/aiosqlite-0.10.0 » rehash
(venv37) jreese@jreese-mbp ~/scratch/aiosqlite-0.10.0 » honesty --help
Traceback (most recent call last):
  File "/Users/jreese/scratch/venv37/bin/honesty", line 5, in <module>
    from honesty.cmdline import main
  File "/Users/jreese/scratch/venv37/lib/python3.7/site-packages/honesty/cmdline.py", line 7, in <module>
    import click
ModuleNotFoundError: No module named 'click'
```

**Comment by jreese** [comment]
Same thing for aiohttp

**Comment by thatch** [comment]
Yeah, there are no tests that actually use the install requires. Should be fixed in 0.1.2 and just pushed a test
